app/src/main/java/com/babylon/wallet/android/presentation/settings/accountsecurity/seedphrases/reveal/RevealSeedPhraseScreen.kt

```diff
@@ -158,6 +162,15 @@ private fun RevealSeedPhraseContent(
                         )
                     }
                 }
+                if (BuildConfig.DEBUG_MODE) {
```

Comment:
I've added debug button to copy whole seed phrase, this is really helpful when trying to input mnemonic quick for testing purposes, for example after backup restoration

Reply:
that's great 
thanks! 🙌🏽